vnext/Microsoft.ReactNative/Modules/ImageViewManagerModule.cpp

```diff
@@ -79,6 +80,28 @@ void ImageLoader::Initialize(React::ReactContext const &reactContext) noexcept {
 }
 
 void ImageLoader::getSize(std::string uri, React::ReactPromise<std::vector<double>> &&result) noexcept {
+  // VALIDATE URI - file:// abuse PROTECTION (P0 Critical - CVSS 7.8)
```

Comment:
What is wrong with file:// here?  Or appx:// or probably a bunch of other things.
If an dev wanted to write a image viewer in RNW, they'd need file://.